.github/workflows/windows-packages.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ windows-2019 ]
+        os: [ windows-2022 ]
```

Comment:
An open question is if we should just change this to use `windows-latest` since that will change as older Windows images are deprecated.